### PR TITLE
Update Font Awesome to 6.2.1

### LINKED ISSUE #291: Upgrade Font Awesome
We are still using Font Awesome 4.  It would be good to upgrade to the latest version.  I think we should stick with using the webfonts version vs the new JS/SVG versions. 

Fontawesome is currently an npm dependency and the css is specified in `angular.json` to be included in the angular build.  This same integration should be maintained for the new version.

Compatibility with v4 should be included as well to ease migration for downstream projects.
https://fontawesome.com/docs/web/setup/host-yourself/webfonts#version-4-compatibility

### PR DESCRIPTION
**Pull request by josie-kaufman-bluehalo**
This fix updates FA to the latest version 6.2.1 from 4.7. Also allows for backwards compatibility for faV4

Updates have been made to icons with naming convention suffix "-o" to reflect new naming conventions.

### CODE REVIEW

**Comment by jrassa** [comment]
What version of node/npm are you using?  `package-lock.json` was changed to an older format.  On install, npm then upgraded back to the current format.  Should be using node 16.* and npm 8.*.  Can you switch versions and rerun `npm install`? The result is a much smaller diff.

**Comment by jrassa** [comment]
> Updates have been made to icons with naming convention suffix "-o" to reflect new naming conventions.

These all should have all had `fa` replaced with `fa-regular` to maintain the outline style.

File: src/styles/ngx-bootstrap/_cards.scss [hunk 1]
```diff
@@ -41,7 +41,7 @@ $card-header-height: 4rem !default;
 
 	&.selected {
 		&:before {
-			font: normal normal normal math.div($card-selected-check-size, 2) FontAwesome; /* stylelint-disable-line */
+			font: var(--fa-font-solid);; /* stylelint-disable-line */
```
Comment (jrassa) [inline src/styles/ngx-bootstrap/_cards.scss]: Need to maintain the specified font-size.  May need to use a separate `font-size` style declaration.

File: src/styles/_selection.scss [hunk 2]
```diff
@@ -139,7 +139,7 @@ input {
 			}
 
 			&:after {
-				font: normal normal normal math.div($checkbox-inner-size, 1) FontAwesome; /* stylelint-disable-line */
+				font: var(--fa-font-solid);
```
Comment (jrassa) [inline src/styles/_selection.scss:142]: Need to maintain the specified font-size.  May need to use a separate `font-size` style declaration.

**Comment by jrassa** [comment]
Sizing of Feedback flyout button is off.  Properly due to size difference with updated arrow icons.  Arrow should be inline with `Feedback`

Screen Shot 2023-01-06 at 9 04 14 AM

**Review by jrassa — CHANGES_REQUESTED** [review]

**Comment by josie-kaufman-bluehalo** [comment]
package-lock.json issue has been fixed.

File: src/styles/ngx-bootstrap/_cards.scss (hunk 1 above)
Comment (josie-kaufman-bluehalo) [reply src/styles/ngx-bootstrap/_cards.scss]: The font size calculation has been moved into a separate line.

File: src/styles/_selection.scss (hunk 2 above)
Comment (josie-kaufman-bluehalo) [reply src/styles/_selection.scss:142]: The font size calculation has been moved into a separate line.

**Comment by josie-kaufman-bluehalo** [comment]
Looking for some help with fixing the fonts of "Firefox" and "Chrome" in the browser recommendation section under the help tab. I think the styling of the text was lost in the upgrade and am unfamiliar with css Pseudo-elements.
Before:
Screen Shot 2023-01-18 at 3 51 18 PM
After fixes: 
Screen Shot 2023-01-18 at 3 50 44 PM

**Comment by jrassa** [comment]
> Looking for some help with fixing the fonts of "Firefox" and "Chrome" in the browser recommendation section under the help tab. I think the styling of the text was lost in the upgrade and am unfamiliar with css Pseudo-elements.

Looks like the fontawesome font was being applied to the text as well, which looked fine with previous version.

In `getting-started-help.component.html`, you just need to separate the text from the icon.

Before:
```
<pre><span class="fa-brands fa-3x fa-firefox"> Firefox</span>    <span class="fa-brands fa-3x fa-chrome"> Chrome</span></pre>
```

After:
```html
<pre>
	<span class="fa-brands fa-3x fa-firefox"></span>
	<span class="ml-3" style="font-size: 3em">Firefox</span>
	<span class="fa-brands fa-3x fa-chrome"></span>
	<span class="ml-3" style="font-size: 3em">Chrome</span>
</pre>
```

**Comment by jrassa** [comment]
Since you will already be making changes there, can we also updated the firefox and chrome icons to use the FA5 styles.

Replace `fa fa-3x fa-firefox` with `fa-brands fa-3x fa-firefox`
Replace `fa fa-3x fa-chrome` with `fa-brands fa-3x fa-chrome`

**Review by jrassa — APPROVED** [review]

**Pull request merged**